README.md

```diff
@@ -61,25 +61,28 @@ The manager looked at the engineer and was not happy.
+This excludes data categorized as being of no value or already publicly available.
 
-EDIT: We are getting a lot of questions about who this applies to, so let's make it clear:
-everyone except:
+In addition, Developers roles (after 1nd stage of Trial period i.e. finishing Trial task#3) are required to set up a Secure Work Environment on their workstations.
+This is both for additional security and improved focus.
 
-- software developers in the first phase of the trial period (where they work on fake/mock data)
-- software developers who work exclusively on open-source projects (that only happened twice, though)
+Setting up Storage Encryption might initially appear daunting, but rest assured that it is fairly straightforward process.
+Our top recommendation is Full Disk Encryption (FDE), a method that encrypts the entire disk for comprehensive protection.
+Our second recommendation is Volume Encryption, a method that encrypts only the volume (/folder) where sensitive data is stored.
+Due to its ease of use, we recommend VeraCrypt by default, for which a simple tutorial can be found [here](https://github.com/reef-technologies/handbook/blob/master/docs/VeraCrypt.md).
+If you want to learn more about available options, please refer to the [Storage Encryption](docs/Storage_Encryption.md) document.
 
-So all staff, including virtual assistants, must have encrypted devices.
-Now that might sound scary, but these days setting up encryption is not that hard - in fact, instructions are provided in this document.
+If you plan to connect your mobile device to any work-related accounts, first make sure that you will be able to provide a documentation that your device is encrypted and is actively supported (i.e. receives regular security updates).
+Most modern Apple and Android devices support encryption, but older devices or ones without configured passcode might not.
 
-Typically, source code is inside a virtual machine which has its image stored on an encrypted volume, but dual boot with full disk encryption is also allowed.
-This practice prevents potential data leaks, i.e., if your computer gets stolen.
+So to sum it up:
 
-You should already have an email account in the reef.pl domain.
-If you qualify for encryption, only ever use this email address (and the associated accounts) on a secure environment (VM/dual boot) - this way, your private and work accounts will never mix access or customer data.
-Docker containers containing client code won't be reachable from the system you use privately.
+- Software Developers during their 1st stage of Trial period are not required to set up Storage Encryption nor Secure Work Environment.
+- Software Developers **after** completing 1st stage of Trial period are required to set up a Secure Work Environment on their workstations.
+- All other roles (e.g. Virtual Assistants) are required to use Storage Encryption (go to [Storage Encryption](docs/Storage_Encryption.md) for more details).
```

Comment:
ok this is super confusing to me. Can we make a section named Storage Encryption, another called Secure Work Environment and above it say who what applies to?

Also, do we want developers in 2nd stage of trial period to set anything up? I don't think we need that, do we?

Reply:
I have changed the stage requirement and added links to these sections